app/views/registrations/new.html.haml

```diff
@@ -1,30 +1,39 @@
-%h4 New Account Sign Up
-= form_with model: @account, url: registration_path do |f|
-  - if @all_error_messages
-    .error-messages
+%div.form-container
+  %h4 New Account Sign Up
+  - if @all_error_messages.present?
+    %div.alert.alert-danger
+      %h4= pluralize(@all_error_messages.count, "error") + " prohibited this account from being saved:"
       %ul
         - @all_error_messages.each do |msg|
           %li= msg
-  %br
-  %p Choose a name & country for your account
-  = f.label :account_name
-  = f.text_field :name
-  %br
-  = f.label :country
-  = f.select :country, options_for_select(ISO3166::Country.countries.map { |c| ["#{c.emoji_flag} #{c.iso_short_name}", c.iso_short_name] }), class: 'country-select'
-  = f.hidden_field :subdomain, value: @subdomain
-  %br
-  %p Create your administrator login
-  = f.fields_for :users do |user_fields|
-    = user_fields.label :admin_name
-    = user_fields.text_field :name
+
+  = form_with model: @account, url: registration_path do |f|
+    %div.form-group
+      = f.label :account_name, class: "form-label"
+      = f.text_field :name, class: "form-control"
+    %div.form-group
+      = f.label :country, class: "form-label"
+      = f.select :country, options_for_select(ISO3166::Country.countries.map { |c| ["#{c.emoji_flag} #{c.iso_short_name}", c.iso_short_name] }), class: 'country-select'
```

Comment:
I think we should fetch Countries selection data from helper methods.

Reply:
Done.